scripts/check-examples.sh

```diff
@@ -6,7 +6,7 @@ pnpm nx run create-swarmion-app:build
 BASE_DIR=$(pwd)
 TEMP_DIR=$(mktemp -d)
 
-for template in swarmion-starter swarmion-full-stack; do
+for template in swarmion-starter swarmion-full-stack swarmion-with-next; do
```

Comment:
We should loop directly on the `examples` folder, something like

```suggestion
for template in examples/*; do
```